### DISCUSSION TITLE
Steam Deck support

### DISCUSSION THREAD

**Discussion by RyanGipson** [AdnanHodzic/displaylink-debian#709] [Ideas]
Hi,

I'll be getting a Steam Deck soon. It is a very capable piece of hardware, but alone it doesn't make a great workstation. I would love to be able to dock it to my DisplayLink based clamshell and use it for portable productivity without needing to wipe it and put Ubuntu or Debian on it.

Steam Deck uses some form of a customized Arch linux.

I'll certainly try to use this when I get it in case it just happens to work well out of the box, but I have doubts about that.

Anyone have further thoughts on this device?

**Comment by RyanGipson** [comment]
Someone wrote up a guide for this at this wiki, if anyone wants to incorporate it into this project in some way.
https://steamdecklinux.wiki/wiki/Guide:Setting_up_DisplayLink

**Reply by devincurrie** [reply]
The weblink doesn't work?
